basalt/autograd/attributes.mojo

```diff
@@ -37,13 +36,12 @@ struct AttributeVector(Sized, Stringable, CollectionElement):
     fn __str__(self) -> String:
         var s: String = "["
         for i in range(self._size):
-            s += str(self._attrs[i])
+            # s += str(self._attrs[i])
```

Comment:
I think it is possible to still have these just pointing it out.

Reply:
Absolutely!